src/meshdb/settings.py

```diff
@@ -248,6 +248,11 @@
     ],
     "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
     "DEFAULT_FILTER_BACKENDS": ["django_filters.rest_framework.DjangoFilterBackend"],
+    "DEFAULT_RENDERER_CLASSES": [
+        "rest_framework.renderers.JSONRenderer",
+        # Removes HTML form (doesn't work with NestedKeyObjectRelatedField)
+        "meshapi.util.drf_renderer.OnlyRawBrowsableAPIRenderer",
```

Comment:
What does this change? Does this change the look of API endpoints in the web browser? Can you show me an example?

Reply:
Sent you a screenshot on slack. This removes the HTML form from the bottom of the page in the browser based API view. The JSON one is still there.